docs/site/tutorials/todo-list/todo-list-tutorial-repository.md

```diff
@@ -87,52 +87,22 @@ export class TodoListRepository extends DefaultCrudRepository<
 
 ### Inclusion of Related Models
 
-To get the related `Todo` objects for each `TodoList`, we can use register a
-custom
-[`InclusionResolver`](https://loopback.io/doc/en/lb4/apidocs.repository.inclusionresolver.html)
-in the `TodoList` repository.
-
-First, add the following import:
-
-```ts
-import {InclusionResolver} from '@loopback/repository';
-```
-
-Next, in the constructor, add the following custom resolver:
+To get the related `Todo` objects for each `TodoList`, we can register the
+inclusion resolver that comes with the
+[`HasManyRepositoryFactory`](https://loopback.io/doc/en/lb4/apidocs.repository.hasmanyrepository.html).
+We need to register this resolver to the repository class, which we can do as
+follows:
 
 ```ts
-export class TodoListRepository extends DefaultCrudRepository</*..*/> {
-  // ...
-  constructor(
-    //db, relation factories setup
-
-    // add the following code to build a custom resolver
-    const todosResolver: InclusionResolver<TodoList, Todo> = async todoLists => {
-      const todos: Todo[][] = [];
-      for (const todoList of todoLists) {
-        const todo = await this.todos(todoList.id).find();
-        todos.push(todo);
-      }
-
-      return todos;
-    };
-    // the resolver needs to be registered before using
-    this.registerInclusionResolver('todos', todosResolver);
-  )
-}
-```
+this.todos = this.createHasManyRepositoryFactoryFor(
```

Comment:
https://github.com/strongloop/loopback-next/issues/3742 this issue makes me wonder that would it be better if we add more descriptive comments, or even a LB4 web link, in the docs and files such as `todo.repository.ts`?

Reply:
Not sure if it's the same comment as @agnes512. but i think it would be good to include the file name that we're editing.